src/components/article-components/ArticleComments.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { api } from '../../api';
+import CommentCard from './CommentCard';
+
+const ArticleComments = ({ articleId }) => {
+	const [comments, setComments] = useState([]);
+
+	useEffect(() => {
+		api.fetchArticleCommentsById(articleId).then(res => {
+			setComments(res);
+		});
+	}, [articleId]);
+
+	return (
+		<section className='comments-container'>
+			<div className='new-comment'>
+				<form>
+					<label htmlFor='newComment'>Write a new Comment:</label>
+					<textarea
+						name='newComment'
+						id='newComment'
+						cols='30'
+						rows='4'
+					></textarea>
+					<button>Submit</button>
+				</form>
+			</div>
+
+			<div className='divider-small'></div>
+
+			<div className='comment-list'>
```

Comment:
You're mapping over a collection of comments here, can you think of an html tag that might be a little more semantic than a div? 
You might want to also have a think about semantic html element you could use for your CommentCard component.

Reply:
no excuse, sorry, this should be a list element with list item elements holding the comment card components, not even sure why i blanked that out, thanks